pkg/services/notification/handler_test.go

```diff
@@ -0,0 +1,94 @@
+package notification
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"openpitrix.io/logger"
+	"openpitrix.io/notification/pkg/config"
+	"openpitrix.io/notification/pkg/pb"
+	"openpitrix.io/notification/pkg/util/pbutil"
+)
+
+func TestCreateNotification(t *testing.T) {
+	if !*TestingEnvEnabled {
```

Comment:
make test will failed. Need to define this in e2e test, make compose-up first, and then make e2e-test, like openpitrix.